js/components/user/form.vue

```diff
@@ -7,24 +7,53 @@ import User from 'models/user';
 
 export default {
     props: {
+        // Edited user
         user: {
             type: User,
             default: function() {return new User();}
-        }
+        },
     },
     data: function() {
-        return {
-            fields: [{
-                    id: 'first_name',
-                    label: this._('First name')
-                }, {
-                    id: 'last_name',
-                    label: this._('Last name'),
-                }, {
-                    id: 'about',
-                    label: this._('About'),
-                }]
-        };
+        let fields = [
+            {
+                id: 'first_name',
+                label: this._('First name'),
+            },
+            {
+                id: 'last_name',
+                label: this._('Last name'),
+            },
+            {
+                id: 'website',
+                label: this._('Website'),
+            },
+            {
+                id: 'about',
+                label: this._('About'),
+            },
+        ]
+        if (this.$root.me.is_admin === true) {
+            fields.push(
+                {
+                    id: 'roles',
+                    label: this._('Roles'),
+                    widget: 'select-input',
+                    multiple: true,
+                    values: [{id: 'admin', name: 'admin'},
+                             {id: 'editor', name: 'editor'},
+                             {id: 'moderator', name: 'moderator'}],
+                    map: function(item) {
```

Comment:
map: item => {return {value: item.name, text: item.name}} works

If I remove the first level of {}, it does not work any more. And if I remove the return, it does not work too, probably because the {{ }} are misinterpreted by vuejs.